src/plans.js

```diff
@@ -503,13 +503,13 @@ function insertVersion (schema, version) {
 }
 
 function fetchNextJob (schema) {
-  return ({ includeMetadata, priority = true } = {}) => `
+  return ({ includeMetadata, priority = true, skipStartAfterNow = false } = {}) => `
```

Comment:
```suggestion
  return ({ includeMetadata, priority = true, ignoreStartAfter = false } = {}) => `
```

Reply:
Looks great.  I'd prefer to rename this to be closer to the original job config property `startAfter`.

Reply:
I've just pushed the rename 👍 Thank you!